app/src/main/java/com/duycuong/weather/data/source/remote/WeatherRemoteDataSource.java

```diff
@@ -0,0 +1,127 @@
+package com.duycuong.weather.data.source.remote;
+
+import android.content.res.Resources;
```

Comment:
ERROR: Unused import: android.content.res.Resources. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports